### ISSUE TITLE
How to loadData by release on pull ?

### ISSUE THREAD

**Issue by atom2ueki** [jsdf/react-native-refreshable-listview#27]
is there a function or way to control load data only after release on pull action ? it need able to working with `minPulldownDistance` so that the data only refresh when hit both conditions

**Comment by jrichardlai** [comment]
I'm interested in this too

**Comment by iplus26** [comment]
+1

**Comment by atom2ueki** [comment]
@jrichardlai @iplus26 
here is my fork, added this feature, hope it helps
https://github.com/atom2ueki/react-native-refreshable-listview

**Comment by iplus26** [comment]
@atom2ueki Thanks for your effort! 
I've checked out your repo but there still are some problems, since I fail to create an issue directly in your repo, I state here: 
* Customised `refreshingIndictatorComponent` don't show up when pulling down to refresh, instead a default indicator shows up. And the customised one doesn't show up until releasing. 
* Every time I scroll the ListView, the default indicator shows up.

**Comment by atom2ueki** [comment]
@iplus26 thanks for your feedback, I'll try to fix it

**Comment by jsdf** [comment]
I believe this is already implemented on master (the default has changed to refresh on release). The version on master will be released in the near future as 2.0.0

**Comment by jsdf** [comment]
If there's any stuff in your fork which you think is worth being merged into the project I'd happily accept a PR

**Comment by iplus26** [comment]
@jsdf Great! Looking forward to the update!

**Comment by atom2ueki** [comment]
@jsdf I need some time to prepare PR, since I saw the current master already give up use `renderHeader`
